include/matx/transforms/reduce.h

```diff
@@ -2082,11 +2082,7 @@ void __MATX_INLINE__ argmax_impl(OutType dest, TensorIndexType &idest, const InT
   MATX_NVTX_START("argmax_impl(" + get_type_str(in) + ")", matx::MATX_NVTX_LOG_API)
 
   cudaStream_t stream = exec.getStream();
-  // example-begin reduce-2
```

Comment:
The docs are still referring to this example, so you'll need to remove it from the docs rst file